### PR TITLE
Marketo updates

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Clarified information related to settings

### CODE REVIEW

**Review by yashasvibajpai — APPROVED** [review]

File: docs/destinations/streaming-destinations/marketo.mdx [hunk 1]
```diff
@@ -64,23 +60,17 @@ If this setting is disabled, events that don't contain the <code class="inline-c
...
+The following settings are associated with the <Link to="#track">`track`</Link> events and require you to first <a href="https://experienceleague.adobe.com/docs/marketo/using/product-docs/administration/marketo-custom-activities/create-a-custom-activity.html">create a Custom Activity in Marketo</a> before sending an event. You also need to approve the activity in order to get the <strong>Custom Activity ID</strong>.
 
 <img src="../../assets/event-stream-destinations/marketo-connection-settings-4.png" />
 
-- **Map events to Marketo Activity ID**: Map the event name to your Marketo custom activity ID.
-- **Map event properties to Marketo property name**: Use this setting to map your `track` event properties to the Marketo fields listed in your custom activity.
-- **Map event name to Marketo primary key**: Map the `track` event name to the Marketo primary key associated with the custom activity.
+- **Map events to Marketo Activity ID**: Use this setting to map the event name in `track` call to Marketo's custom activity ID. For example, you can map a **Product Clicked** event to a custom activity ID **100001** associated with the **Product wishlisted** custom activity.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/marketo.mdx:67]:
```suggestion
- **Map events to Marketo Activity ID**: Use this setting to map the event name in `track` call to Marketo's custom activity ID. For example, you can map a **Product Clicked** event to a custom activity ID **100001** associated with the **Product Wishlisted** event.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/marketo.mdx:67]: Currently, it the term "custom activity" feels repetitive and unnecessary - hence the above change.

Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/marketo.mdx:67]:
```suggestion
- **Map events to Marketo Activity ID**: Use this setting to map the event name in `track` call to Marketo's custom activity ID. For example, you can map a **Product Clicked** event to a custom activity with the name **Product wishlisted** and ID **100001**.
```

File: docs/destinations/streaming-destinations/marketo.mdx [hunk 2]
```diff
@@ -130,7 +120,7 @@ For the rest of the fields that you want to sync with Marketo, you can create a
 You can use the <Link to="/event-spec/standard-events/track/">`track`</Link> events to register custom activities in Marketo. 
 
 <div class="warningBlock">
-Make sure you have already <a href="https://experienceleague.adobe.com/docs/marketo/using/product-docs/administration/marketo-custom-activities/create-a-custom-activity.html">created the custom activities in Marketo</a> before sending the events. You also need to approve the activity in order to get the <strong>Custom Activity ID</strong>.
+Make sure to <a href="https://experienceleague.adobe.com/docs/marketo/using/product-docs/administration/marketo-custom-activities/create-a-custom-activity.html">create the custom activities in Marketo</a> before sending the events. Also, approve the activity in order to get the <strong>Custom Activity ID</strong>.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/marketo.mdx:123]: "Also approve the activity in order to get the Custom Activity ID"

how does one do this? Is it using the **Map events to Marketo Activity ID** field or in the Marketo dashboard? 

@ishachopra0401 @yashasvibajpai

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**